include/dlaf/lapack/tile.h

```diff
@@ -220,6 +395,28 @@ void assertExtendInfo(F assertFunc, cusolverDnHandle_t handle, CusolverInfo<T>&&
 }
 }
 
+template <class T>
+dlaf::BaseType<T> lange(cusolverDnHandle_t, const lapack::Norm norm, const Tile<T, Device::GPU>& a) {
+  static_assert(sizeof(T) == 0, "lange is unimplemented for Backend::GPU");
```

Comment:
Yeah, that occurred to me as well. I left it like this because it needs that dummy dependency on `T` to not make it fail even when not instantiated. So it'd have to look like this:

```
#define DLAF_UNIMPLEMENTED_STATIC(dummy_dependency, error_message) static_assert(sizeof(dummy_dependency) == 0, error_message)
/// and then
DLAF_UNIMPLEMENTED_STATIC(T, "lange is unimplemented");
```

I found it slightly ugly but it's not too bad. I can also make it a runtime error with `DLAF_UNIMPLEMENTED`.

Reply:
A minor question about this point: Why we need to define a dummy function that fails to compile? Wouldn't it better to not declare and define a function at all?

Reply:
That would also be fine. The dummy function just gives you a better error message, but it'll fail to compile in either case of course.

Reply:
I've added the `DLAF_STATIC_UNIMPLEMENTED` macro. @albestro had a very good idea though, which is to declare them as `= delete` instead. I think the error messages will look similar. The only downside is that with `= delete` we don't get to attach any contextual information about _why_ the function is deleted.